AirCasting/Settings/BackendURLValidator.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  BackendURLValidator.swift
+//  URLValidation
+//
+//  Created by Pawel Gil on 25/06/2021.
+//
```

Comment:
let's be consistent with those header comments (btw. I hate those and would rather not have them at all ;) ) and put something this here:
```Swift
// Created by Lunar on 25/06/2021.
//
```
AFAIK this is the current meta ;)